libs/remix-lib/src/web3Provider/web3VmProvider.ts

```diff
@@ -155,8 +155,8 @@ export class Web3VmProvider {
     const status = data.execResult.exceptionError ? 0 : 1
     this.txsReceipt[this.processingHash].status = `0x${status}`
 
-    if (data.createdAddress) {
-      const address = hexConvert(data.createdAddress)
+    if (data.createdAddress && data.createdAddress.buf) {
+      const address = hexConvert(data.createdAddress.buf)
```

Comment:
fyi you can use address.toString() to get the 0x-prefixed hex string of the address.